RiotSwiftUI/Modules/Settings/ChangePassword/ChangePasswordModels.swift

```diff
@@ -0,0 +1,65 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+// MARK: View model
+
+enum ChangePasswordViewModelResult {
+    /// Submit with old and new passwords and sign out of all devices option
+    case submit(String, String, Bool)
```

Comment:
It might be nice to provide labels for these associated types as a hint for which is the old and which is the new password.

Reply:
Yep, good idea